docs/getting_started/using_modin/using_modin_locally.rst

```diff
@@ -0,0 +1,124 @@
+=====================
+Using Modin Locally
+=====================
+
+.. note:: 
+  | *Estimated Reading Time: 5 minutes*
+
+..   | You can follow along this tutorial in a Jupyter notebook `here <hhttps://github.com/modin-project/modin/tree/master/examples/tutorial/tutorial_notebooks/introduction/quickstart.  ipynb>`. 
+
+In our quickstart example, we have already seen how you can achieve considerable 
+speedup from Modin, even on a single machine. Users do not need to know how many cores their system has, nor do they need to specify how to distribute the data. In fact, 
+users can **continue using their existing pandas code** while experiencing a 
+considerable speedup from Modin, even on a single machine. 
+
+To use Modin on a single machine, only a modification of the import statement is needed. Once you've changed your import statement, you're ready to use Modin
+just like you would pandas, since the API is identical to pandas.
+
+.. code-block:: python
+
+  # import pandas as pd
+  import modin.pandas as pd
+
+**That's it. You're ready to use Modin on your previous pandas workflows!** 
+
+Optional Configurations
+-------------------------
+
+When using Modin locally on a single machine or laptop (without a cluster), Modin will 
+automatically create and manage a local Dask or Ray cluster for the executing your 
+code. So when you run an operation for the first time with Modin, you will see a 
+message like this, indicating that a Modin has automatically initialized a local 
+cluster for you:
+
+.. code-block:: python
+
+  df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+
+.. code-block:: text
+
+   UserWarning: Ray execution environment not yet initialized. Initializing...
+   To remove this warning, run the following python code before doing dataframe operations:
+
+    import ray
+    ray.init()
+
+ If you prefer to use Dask over Ray as your execution backend, you can use the 
+ following code to modify the default configuration:
+
+.. code-block:: python
+
+   import modin
+   modin.config.Engine.put("Dask")
+
+.. code-block:: python
+
+   df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+
+
+.. code-block:: text
+
+   UserWarning: Dask execution environment not yet initialized. Initializing...
+   To remove this warning, run the following python code before doing dataframe operations:
+
+      from distributed import Client
+
+      client = Client()
+
+Finally, if you already have an Ray or Dask engine initialized, Modin will 
+automatically attach to whichever engine is available. If you are interested in using 
+Modin with OmniSci engine, please refer to :doc:`these instructions </developer/using_omnisci>`. For additional information on other settings you can configure, see 
+:doc:`this page </flow/modin/config>` for more details.
+
+.. note:: 
+   Note that Modin can not be used in conjunction with other distribution mechanism. 
```

Comment:
This is a highly general phrase. Ray can be used with Dask, for example, if Ray and Dask utilize enough resources for their execution. I would rephrase this phrase a bit and more focused on python multiprocessing. Also, I think this should be put in troubleshooting page.

Reply:
I moved this to the troubleshooting section and specified that this is only for python multiprocessing. I can not actually reproduce any error or warning when using Modin on Ray and python multiprocessing together, so I'm not sure what this is referring to.